internal/mode/static/manager.go

```diff
@@ -144,7 +144,7 @@ func StartManager(cfg config.Config) error {
 		return fmt.Errorf("cannot clear NGINX configuration folders: %w", err)
 	}
 
-	var processHandler = &ngxruntime.ProcessHandlerImpl{}
+	processHandler := &ngxruntime.NewProcessHandlerImpl{}
```

Comment:
@kate-osborn did update, is this fine for you ?

Reply:
Almost:
```suggestion
	processHandler := ngxruntime.NewProcessHandlerImpl(os.ReadFile, os.Stat)
```